upgrade/scripts/upgrade/prerequisites.sh

```diff
@@ -1339,41 +1328,11 @@ state_recorded=$(is_state_recorded "${state_name}" "$(hostname)")
 if [[ ${state_recorded} == "0" ]]; then
   echo "====> ${state_name} ..." | tee -a "${LOG_FILE}"
   {
-    # make sure cray-nls-postgres is healthy
```

Comment:
No need to wait for cray-nls-postgres. We restart it and wait for it to come up in FIX_POSTGRES.